packages/client/hmi-client/src/types/common.ts

```diff
@@ -215,13 +215,20 @@ export interface ChartSettingSensitivityOptions {
 	selectedInputVariables: string[];
 	timepoint: number;
 	chartType: SensitivityChartType;
+	method: SensitivityMethod;
 }
 
 export enum SensitivityChartType {
 	SCATTER = 'scatter',
 	HEATMAP = 'heatmap'
 }
 
+export enum SensitivityMethod {
+	TIMEPOINT = 'value at timepoint',
+	MAX = 'peak value',
+	ARGMAX = 'peak timepoint'
+}
```

Comment:
This enum feels a bit weird, in that the labels are meaningful but the enums themselves are a bit ambiguous.
Why not just:

```
export enum SensitivityMethod {
	TIMEPOINT = 'timepoint',
	PEAK_VALUE = 'peak value',
	PEAK_TIMEPOINT = 'peak timepoint'
}

```